packtools/sps/validation/alternatives.py

```diff
@@ -58,6 +102,10 @@ def validation(self, parent_children_dict=None):
         if not parent_children_dict:
             raise ValidationAlternativesException("Function requires dict of parent tag (key) and child tags list (value)")
```

Comment:
@Rossi-Luciano No lugar de `"Function requires dict of parent tag (key) and child tags list (value)"` use `f"The element '{key}' is not configured to use 'alternatives'. Provide alternatives parent and children"`

Reply:
OK, @robertatakenaka